database/migrations/2020_09_10_082029_add_temp_password_to_users_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddTempPasswordToUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->string('temp_password')->nullable()->default(null);
+        });
+
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->dropColumn('temp_password');
+        });
+    }
+}
```

Comment:
This one is no need anymore and so I removed it. If it blocks to rollback, just remove it from the migration table.

Reply:
I added this one because it was present in migrations table. As you mentioned not needed anymore then will remove from migrations table.